deployment/kustomizations/helm-values-qdrant_0.8.5.yaml

```diff
@@ -0,0 +1,234 @@
+replicaCount: 1
+
+image:
+  repository: docker.io/qdrant/qdrant
+  pullPolicy: IfNotPresent
+  tag: "v1.9.1"
```

Comment:
Could we use `v1.7.4` ? Just to align with the version we currently use in `docker-compose`.
https://github.com/Canner/WrenAI/blob/a60d1be585d04f7df5eb9c9dc001c9f48cea31f7/docker/docker-compose.yaml#L61

Reply:
this might be not a good idea to stretch the app and the helm versions so far apart. Testing is needed

Reply:
I see. @cyyeh mentioned that using the latest version of qdrant might cause issues. Maybe we could give it a try ?

Reply:
Sure, we could give it a try and see.